src/workloads/execution/CumulativeWindows.yml

```diff
@@ -0,0 +1,157 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    # Choose a collection large enough such that a sort will trigger a spill to disk.
```

Comment:
What do you mean here?  The queries don't specify `allowDiskUse: true`, so they would fail rather than spill.

I think we also don't expect anything to use much space: the sort should use the index, and the [unbounded, current] window should only need a small, constant amount of space.

Reply:
Yes that's right. This comment says that without the index we would spill, which would result in failure for these tests.

Reply:
Oh, got it, thanks.